Assets/HotmaxEditorVR/objectSelect.cs

```diff
@@ -85,12 +84,14 @@ void Update()
     void ShootLaserFromTargetPosition(Vector3 targetPosition, Vector3 direction, float length)
     {
         Ray ray = new Ray(targetPosition, direction);
-        RaycastHit raycastHit;
+        RaycastHit hit;
         laserEndPosition = targetPosition + (length * direction);
 
-        if (Physics.Raycast(ray, out raycastHit, length))
+        if (Physics.Raycast(ray, out hit, length))
         {
-            laserEndPosition = raycastHit.point;
+            laserEndPosition = hit.point;
+            //TODO: this should go somewhere more explicit
+            _stateManagerMutatorRef.SET_RAYCAST_HIT_INFO(hit.point, hit.collider.gameObject);
```

Comment:
Shouldn't this be down in your select function?

Reply:
you are sending a mutator out every frame right now

Reply:
how else can I update the sliders position...?

Reply:
if it is in select it will only fire on click down

Reply:
we'll figure something out - this is a pretty tremendous overhead for handling a slider in a menu somewhere.

Reply:
i'll loop back around

Reply:
yup, there has to be something listening to the raycast hit.point